osm2gtfs/core/stops.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+
+import attr
+
+
+@attr.s
+class Stop(object):
+
+    osm_id = attr.ib()
+    osm_type = attr.ib()
```

Comment:
Why do we need an `osm_type`? What values can this have?

Reply:
It can be either `node`, `way` or `relation`. Depending on the type different behaviours might be necessary. E.g for a `way` you need to find the center node, for a `relation` you might look into the members.

Reply:
Another reason is also related to #94: `osm_id` is used for different id spaces, depending on `osm_type`. Both together can identify an object in OSM, not only `osm_id`. For the id of the GTFS it is recommended to combine those two in any way.